src/config/musl.md

```diff
@@ -0,0 +1,94 @@
+# musl
+
+[musl](https://musl.libc.org/) is a libc implementation which strives to be
+lightweight, fast, simple, and correct.
+
+Void officially supports musl by using it in its codebase for all target
+platforms except i686. Additionally, all the packages in our official
+repositories are available with musl-linked binaries in addition to their glibc
+counterparts.
+
+Currently, there are nonfree and debug subrepositories for musl, but no multilib
+subrepo.
+
+Generally speaking, a musl based system is a good choice for servers, embedded
+solutions and old PCs, but it may not be worth the effort to use on a modern
+desktop computer.
```

Comment:
this is a subjective opinion and shouldn't be present in the handbook either

i don't see what makes musl any more suited for servers than desktops

on old PCs the claim makes some sense because of lower memory footprint, but yeah, I wouldn't bother

Reply:
Removed.